datahub-frontend/conf/application.conf

```diff
@@ -39,6 +39,11 @@ play.http.server.akka.max-header-count = ${?DATAHUB_AKKA_MAX_HEADER_COUNT}
 play.server.akka.max-header-value-length = 8k
 play.server.akka.max-header-value-length = ${?DATAHUB_AKKA_MAX_HEADER_VALUE_LENGTH}
 
+# pac4j configuration
+# default to PlayCookieSessionStore to keep datahub-frontend's statelessness
+pac4j.sessionStore.provider= "PlayCookieSessionStore"
+pac4j.sessionStore.provider= ${?PAC4J_SESSIONSTORE_PROVIDER}
```

Comment:
Amazing! I'm wondering if we could document this - at least in our FAQ or debugging guide...

Reply:
any idea where I should add this in?

Reply:
For now, how about just adding an FAQ here- https://datahubproject.io/docs/debugging 

@RyanHolstien is working on revamping our deployment docs, perhaps we can also add a note in there once they are live. 

Thanks again, @neojunjie !

Reply:
got it. will add them in 
working on the other pr. =(

Reply:
@jjoyce0510 added them in debugging.md. Anything else to add?

Reply:
I think that works for now.. Thank you!